src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/WfsXmlQuery/WfsXmlQuery.xaml.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Diagnostics;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.WfsXmlQuery
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Load WFS with XML query",
+        "Layers",
+        "Load a WFS feature table using an XML query.",
+        "")]
+    public partial class WfsXmlQuery
+    {
+        // Constants for the service URL, table name, and the query.
```

Comment:
Is there any WFS documentation hyperlinks that we can point the user to on how to construct such an XML string? Knowing this exact syntax is essential to making this sample work. How was it obtained?